docs/contributing.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Contributing
+linktitle: Contributing
+description: "A guide for how to contribute to stack via issues, pull requests etc."
+categories: []
+keywords: ['stack', 'docs', 'wordpress', 'kubernetes', 'contributing', 'development', 'issues']
+draft: false
+aliases: []
+slug: contributing
+toc: true
+related: true
+---
+
+## Issues
+Issues are being tracked at https://github.com/presslabs/stack/issues.  
+They can range from bug reports to questions about developoment, installation process and other stuff related to the project.
+
+We don't run a code of conduct, but we trust you to be civil.
```

Comment:
Not necessary in the context of `Issues`. Maybe we can have something like an intro or short description.